cmd/containerd/builtins_linux.go

```diff
@@ -22,6 +22,7 @@ import (
 	_ "github.com/containerd/containerd/runtime/v1/linux"
 	_ "github.com/containerd/containerd/runtime/v2"
 	_ "github.com/containerd/containerd/runtime/v2/runc/options"
+	_ "github.com/containerd/containerd/snapshots/devmapper"
```

Comment:
Will leave it to maintainers to decide but I'd like to see this similar to btrfs and cri where build options can disable it, otherwise people will end up rewriting the `main.go` just to avoid initialization of built ins they don't care about

Reply:
Thanks for pointing this out, I'll think about it for a bit

Reply:
@crosbymichael a setting to skip plugin init would be nice instead of doing this stuff everywhere, it'll just get cumbersome to manage over time.

Reply:
Yes, can we break this out into a file with a build tag just like btrfs and others?

Reply:
Done!